src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -404,6 +404,7 @@ def load_arguments(self, _):
         c.argument('vault_name', vault_name_type, id_part='name')
         c.argument('log_point_in_time', options_list=['--log-point-in-time'], help="""Specify the point-in-time (in UTC) which will be restored.""")
         c.argument('rp_name', rp_name_type)
+        c.argument('target_instance_name', options_list=['--target-instance-name'], help="""Specify the target instance name for the restore operation.""")
```

Comment:
```suggestion
        c.argument('target_instance_name', help="""Specify the target instance name for the restore operation.""")
```

Reply:
`--target-instance-name` is the default option name which does not need explicit duplicate definitions